src/NBetting.Web/wwwroot/js/app/home/components/myTournamentsComponent.js

```diff
@@ -0,0 +1,20 @@
+﻿function MyTournamentsController(tournamentService) {
+    var vm = this;
+    vm.myTournaments = [];
+
+    function getData() {
+        vm.myTournaments = tournamentService.getMyTournaments();
+    }
+
+    this.$onInit = getData;
```

Comment:
Nie widzę, sensu aby zrobić to w taki sposób. Nie lepiej $onInit = function(){}.